web-components/src/components/inputs/validation.ts

```diff
@@ -117,3 +118,23 @@ export function validatePrice(
   }
   return;
 }
+
+/**
+ * Checks if the given string is an ETH address
+ *
+ * @method isValidEthAddress
+ * @param {String} address the given HEX adress
+ * @return {Boolean}
+ */
+const isValidEthAddress = (address: string): boolean => {
```

Comment:
Should These functions belong to registry instead?

Reply:
Yes, maybe a validation module as @victordelval mentioned, but I will save that for later.